getting-started/setup-building.rst

```diff
@@ -446,6 +446,155 @@ used in ``python.sh``:
 .. _wasmtime: https://wasmtime.dev
 .. _WebAssembly: https://webassembly.org
 
+iOS
+---
+
+Compiling Python for iOS requires a macOS machine, on a recent version of macOS,
+running a recent version of Xcode. Apple expects developers to keep their
+operating systems and tools up-to-date; if your macOS version is more than one
+major release out of date, or your Xcode version is more than a couple of minor
+versions out of date, you'll likely encounter difficulties. It is not possible
+to compile for iOS using Windows or Linux as a build machine.
+
+A complete build for Python on iOS requires compiling CPython 4 times: once for
+macOS; then once for each of the three underlying platforms used by iOS:
+
+* An ARM64 device (an iPhone or iPad);
+* An ARM64 simulator running on a recent macOS machine; and
+* An x86_64 simulator running on older macOS machine.
+
+The macOS build is required because building Python involves running some Python
+code. On a normal desktop build of Python, you can compile a Python interpreter
+and then use that interpreter to run Python code. However, the binaries produced
+for iOS won't run on macOS, so you need to provide an external Python
+interpreter. From the root of a CPython code checkout, run the following:
+
+.. code-block:: shell
+
+    $ ./configure --prefix=$(pwd)/cross-build/macOS
+    $ make -j4 all
+    $ make install
```

Comment:
```suggestion
interpreter. From the root of a CPython code checkout, run the following::

   $ ./configure --prefix=$(pwd)/cross-build/macOS
   $ make -j4 all
   $ make install
```

The page already has `console` as default highlight.  Also indentation is 3 spaces.